test/dal_test_case.exs

```diff
@@ -122,4 +122,22 @@ defmodule DALTest.Case do
 
     %{genres: genres}
   end
+
+  def upsert_genre(_) do
```

Comment:
Is this used anywhere?

Reply:
No, I wasn't sure if accompanying tests should take place with this.